scripts/test/wallet/integration.sh

```diff
@@ -31,27 +31,30 @@ rm -rf $tmpSecrets
 stack exec -- cardano-keygen --system-start 0 generate-keys-by-spec --genesis-out-dir $tmpSecrets
 
 # run cluster
-# TODO (akegalj): move to integration/Main.hs
-# WALLET_DEBUG flag is needed so that wallet is launched without TLS. We could also use --no-tls, but this involves changes to the underlying script. See https://iohk.myjetbrains.com/youtrack/issue/CT-7#comment=93-17848
 echo "Starting local cardano cluster..."
 tmux new-session -s $sessionName -d "WALLET_DEBUG=1 scripts/launch/demo-with-wallet-api.sh"
 
 # wait until cluster is fully up and running
-echo "Waiting 140 seconds until local cluster is ready..."
-sleep 140s
+echo "Waiting 40 seconds until local cluster is ready..."
+sleep 40s
```

Comment:
40s doesn't work on my machine (80s does). It will be run by our CI which would probably handle 40 good enough